packages/popover/src/popover.css

```diff
@@ -10,3 +10,52 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 @import './spectrum-popover.css';
+
+:host([placement*='bottom'][open]) {
+    /* .spectrum-Popover--bottom.is-open */
+    max-height: calc(
+        100% -
+            var(
+                --spectrum-dropdown-flyout-menu-offset-y,
+                var(--spectrum-global-dimension-size-75)
+            )
+    );
+}
+:host([placement*='top'][open]) {
+    /* .spectrum-Popover--top.is-open */
+    margin-top: var(
+        --spectrum-dropdown-flyout-menu-offset-y,
+        var(--spectrum-global-dimension-size-75)
+    );
+    max-height: calc(
+        100% -
+            var(
+                --spectrum-dropdown-flyout-menu-offset-y,
+                var(--spectrum-global-dimension-size-75)
+            )
+    );
+}
+:host([placement*='right'][open]) {
+    /* .spectrum-Popover--right.is-open */
+    max-width: calc(
+        100% -
+            var(
+                --spectrum-dropdown-flyout-menu-offset-y,
```

Comment:
Weird that we still use the `y` offset for right and left, but we are doing that seems to be how [spectrum-css is doing it](https://github.com/adobe/spectrum-css/blob/master/components/overlay/index.css#L14)